src/modules/urlParamsPreValidation.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const urlParamsPreValidation = (textToConvert, targetCase) => {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const caughtErrors = {
+    errors: [],
+  };
```

Comment:
I think it is better to return just array

Reply:
fixed